### DISCUSSION TITLE
충돌해제시킨 레이어끼리 스크립트상에서 충돌처리를 합니다. [버그]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#200] [Issue Report (Legacy)]
유니티에서는 문제가 없지만 모바일 환경에서는 프로젝트 설정의 피직스 설정에서 20~27번으로 셋팅된 레이어끼리 충돌해제를 해두었는데 충돌을 합니다.
물리충돌은 해제가 되는것 같은데 OnTriggerEnter부분을  호출해요.

유니티에서는 잘 적용되서 스크립트를 호출하지 않음
모바일에서는 충돌해제된 레이어끼리도 스크립트를 호출함

**Comment by Joyounwoo** [comment]
Project Settings -> Physics의 Layer Collision Matrix 값이 제페토 앱 상에선 반영이 안되는 것 같습니다.

그렇기에 별도로 월드 로딩 직후에 제페토 스크립트에서
`Physics.IgnoreLayerCollision([LayerNumber], [LayerNumber])`
메소드를 통해 레이어 간 충돌을 해제하시면 됩니다.

**Reply by x4cf** [reply]
방법만 있다면 좋아요 :D

**Comment by x4cf** [comment]
고... 쳐... 줘...
<img width="1281" alt="스크린샷 2022-01-13 오전 10 05 40" src="https://user-images.githubusercontent.com/90679926/149247542-83fe97a4-a76d-4393-93d6-aa5a86e09fc7.png">

**Comment by cheolhui-lee** [comment]
안녕하세요.
Physics Layer 설정이 패키지 될 때 포함되지 않기 때문에 현재는 지원이 안되고 있습니다.
하지만 해당 내용 적용 계획에 있으니 반영이 되면 다시 공유 드릴 수 있도록 하겠습니다.
